esmvalcore/preprocessor/_derive/__init__.py

```diff
@@ -52,11 +52,11 @@ def get_required(short_name, project):
         List of dictionaries (including at least the key `short_name`).
 
     """
-    if short_name not in ALL_DERIVED_VARIABLES:
+    if short_name.lower() not in ALL_DERIVED_VARIABLES:
```

Comment:
The wind variable in a recipe would be spelled `sfcWind`, so the `.lower()` part would be required to marry the input from the recipe to the preprocessor.

Reply:
`short_name` comes from the CMOR table, so that will always be lower case - as I recall, it doesn't come from the recipe

Reply:
Interesting, from the quick look, the most variables indeed are spelled in lowercase, so wind seems to be an exception:  (see e.g.  https://github.com/ESMValGroup/ESMValCore/blob/bdad237ded5e42e915ce05ca7d284b10eef30eb4/esmvalcore/cmor/tables/cmip6/Tables/CMIP6_day.json#L431C18-L431C18). That being said, as I mentioned in the discussion, I am happy to rename it as a `der_sfcwind` highlighting that this is a wind derived from `uas` and `vas` and not the native `sfcWind` :-)

Reply:
sorry to be pedantic, Liza, but as you can see from the table `"standard_name": "wind_speed"` - I am fairly sure all standard names need be lower case in CMOR format, but @zklaus (our resident CMOR/standards specialist) can correct me if I is wrong :smiley:

Reply:
and a massive :man_facepalming: to me since all this time I kept yapping about `standard_name` when in fact Liza is fixing a problem with `short_name` - sorry, Liza, my bad, and a slip of attention!

Reply:
I even wrote `short_name` once in a comment, was still thinking of `standard_name` :rofl: